lib/rubocop/options.rb

```diff
@@ -495,6 +499,14 @@ module OptionsHelp
       disable_uncorrectable:            ['Used with --autocorrect to annotate any',
                                          'offenses that do not support autocorrect',
                                          'with `rubocop:todo` comments.'],
+      first_line:                       ['Used with --autocorrect to specify',
```

Comment:
I'm wondering if `start_line` and `end_line` won't be slightly better terminology here.